mcs/class/corlib/Mono/Runtime.cs

```diff
@@ -147,7 +147,8 @@ static void SendExceptionToTelemetry (Exception exc)
 			SendMicrosoftTelemetry (payload_str, portable_hash, unportable_hash);
 		}
 
-		static void EnableMicrosoftTelemetry (string appBundleID_str, string appSignature_str, string appVersion_str, string merpGUIPath_str, string eventType_str, string appPath_str)
+		// All must be set except for configDir_str
+		static void EnableMicrosoftTelemetry (string appBundleID_str, string appSignature_str, string appVersion_str, string merpGUIPath_str, string eventType_str, string appPath_str, string configDir_str = null)
```

Comment:
This won't work, reflection requires explicitly stating the argument is missing

Reply:
I was afraid that was the case, but the tests on my end seemed to be happy with it. If that's not a stable way to access the method, I'd like to avoid it though. 

I previously made an EnableMicrosoftTelemetryNew and an EnableMicrosoftTelemetry, and used the "New" one in my tests, but @luhenry suggested using the default argument syntax instead. 

I'll go ahead with pushing that previous fix if that's preferred.

Reply:
``` 
 		// All must be set except for configDir_str
-		static void EnableMicrosoftTelemetry (string appBundleID_str, string appSignature_str, string appVersion_str, string merpGUIPath_str, string eventType_str, string appPath_str, string configDir_str)
+		static void EnableMicrosoftTelemetryNext (string appBundleID_str, string appSignature_str, string appVersion_str, string merpGUIPath_str, string eventType_str, string appPath_str, string configDir_str)
 		{
 			if (RuntimeInformation.IsOSPlatform (OSPlatform.OSX)) {
 				using (var appBundleID_chars = RuntimeMarshal.MarshalString (appBundleID_str))
@@ -165,6 +165,11 @@ namespace Mono {
 				throw new PlatformNotSupportedException("Merp support is currently only supported on OSX.");
 			}
 		}
+
+		static void EnableMicrosoftTelemetry (string appBundleID_str, string appSignature_str, string appVersion_str, string merpGUIPath_str, string eventType_str, string appPath_str)
+		{
+			EnableMicrosoftTelemetryNext (appBundleID_str, appSignature_str, appVersion_str, merpGUIPath_str, eventType_str, appPath_str, null);
+		}
```